src/components/Candidate.js

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { useParams } from 'react-router-dom'
+import {
+  Grid,
+  GridContainer
+} from '@trussworks/react-uswds';
+import Table from './Table';
+import '../css/candidate.scss';
+
+const Candidate = () => {
+  const [candData, setCandData] = useState({})
+  const [contData, setContData] = useState([])
+  let { candidate } = useParams()
+
+  useEffect(() => {
+    const fetchContributions = () => {
+      const url = `/api/candidates/` + candData.sboe_id + `/contributions`;
+      fetch(url)
+        .then((res) => res.json())
+        .then((json) => setContData(json.data))
+    }
+
+    const fetchData = () => {
+      const url = `/api/search/candidates/` + encodeURIComponent(candidate)
+      fetch(url)
+        .then((res) => res.json())
+        .then((json) => setCandData(json.data[0]))
+        .then(() => fetchContributions())
+    }
+    
+    fetchData();
+  }, [candidate, candData.sboe_id])
```

Comment:
Would you consider making these fetch calls `async`? It might read a little bit more clearly

```suggestion
  useEffect(() => {
    const fetchContributions = async () => {
      const url = `/api/candidates/` + candData.sboe_id + `/contributions`;
      const response = await fetch(url)
      const { data } = await response.json()
      setContData(data)
    }

    const fetchData = async () => {
      const url = `/api/search/candidates/` + encodeURIComponent(candidate)
      const response = await fetch(url)
      const { [data] } = await response.json()
      setCandData(data)
    }
    
    await fetchData();
    await fetchContributions();
  }, [candidate, candData.sboe_id])
```

Reply:
Good call, definitely more readable!